timelock-server/src/test/java/com/palantir/atlasdb/timelock/config/AbstractTimelockServerConfigDeserializationTest.java

```diff
@@ -48,10 +44,18 @@
         OBJECT_MAPPER.registerModule(new Jdk8Module());
     }
 
+    protected AbstractTimelockServerConfigDeserializationTest() {}
+
+    public abstract String getConfigFileName();
+
+    public abstract void assertAlgorithmConfigurationCorrect(TimeLockAlgorithmConfiguration configuration);
+
     @Test
     public void canDeserializeTimeLockServerConfiguration() throws IOException {
+        File testConfigFile = new File(
+                PaxosConfigDeserializationTest.class.getResource(getConfigFileName()).getPath());
```

Comment:
nit: It seems a bit unusual that the `TimestampBoundStoreConfigDeserializationTest` would use the `getResource()` method of the `PaxosConfigDeserializationTest` - I think this should just be this class (i.e. `AbstractTimelockServer...Test.class.getResource...`)

Reply:
oops, yes.